music_assistant/controllers/player_queues.py

```diff
@@ -1168,25 +1269,34 @@ async def _load_item(
                     album.provider,
                 )
             ):
-                queue_item.media_item.album = library_album
+                if not isinstance(library_album, Album):
```

Comment:
Do we really need to guard this? Have there been occurrences where the library returns unexpected media types?

Reply:
Not that I'm aware. Switched to an assert for mypy